cmd/ltctl/main.go

```diff
@@ -28,12 +29,20 @@ func RunCreateCmdF(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to create terraform engine: %w", err)
 	}
 
-	initData := config.DBDumpURI == ""
-	err = t.Create(initData)
+	extAgent, err := ssh.NewAgent()
```

Comment:
Curious, is there any advantage in passing this agent around vs creating it inside these methods (`Create` and `PostProcessDatabase`)?